static/src/js/components/OrderProgressList/__tests__/OrderProgressItem.test.js

```diff
@@ -1,57 +1,121 @@
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
 
-import { Badge, ProgressBar } from 'react-bootstrap'
+import { render, screen } from '@testing-library/react'
+
 import { OrderProgressItem } from '../OrderProgressItem'
 
-import { retrievalStatusPropsEsi } from '../../OrderStatus/__tests__/mocks'
+import {
+  retrievalStatusPropsEsi,
+  retrievalStatusPropsSwotOrder,
+  retrievalStatusPropsHarmonyOrder
+} from '../../OrderStatus/__tests__/mocks'
```

Comment:
It feels weird to borrow mocks from another component. Data needed for these tests should be located in the same place as these tests. 

If this test wasn't reusing data designed for another component we wouldn't need the additional logic to dig into the test data to pull out only the order this test needs.